infrastructure/stacks/uec-developer-role/uec-rw-data-services.json

```diff
@@ -28,6 +28,7 @@
                 "redshift:DescribeEvents",
```

Comment:
A bit off topic, but I didn't realise we were using redshift? Isn't that a data warehousing component?

Doesn't the 'VisualEditor2' policy override this one here anyway?

Reply:
ooh good spot .. the json was lifted from the console so not quite sure how the redundant redshift perms got included, I will edit it down to just use redshift.* ... I think as this is intended for use by devs in dev account only I think we included resources we might want to play with in future.